### ISSUE TITLE
Add guidance for alt text

### ISSUE THREAD

**Issue by aleesteele** [the-turing-way/the-turing-way#3268]
### Summary

The Turing Way uses many images in its guides, as well as shares photos in social channels like Twitter, Mastodon & Slack. Having a set of guidance related to how to write alt text for different types of images would be valuable, whether that collects existing resources, writes separate guidance for our community handbook, or otherwise.

### What needs to be done?

- [ ] Collect citations and existing guidance for writing alt text
- [ ] Draft text for Community handbook
- [ ] Review & edit for publication

### Who can help?

* Anyone interested in accessibility of and within The Turing Way!

---

**Comment by tobyhodges** [comment]
Not sure if it is so relevant here, but I love this article about alt-text for data visualisations: https://medium.com/nightingale/writing-alt-text-for-data-visualization-2a218ef43f81

**Comment by BrainonSilicon** [comment]
AIM RSF Gave a short workshop/presentation where we distilled 5 key points of guidance. Google Slides presentation link is [here](https://docs.google.com/presentation/d/1x3nDhrekk-phSDBnOIE6e8l9Bgo4zAoYq8y7HwxnOkc/edit?usp=sharing).

**Comment by aleesteele** [comment]
Thank you @BrainonSilicon and @tobyhodges!

**Comment by PatriLoto** [comment]
Hi, this resource from W3C could be useful: https://www.w3.org/WAI/tutorials/images/

**Comment by PatriLoto** [comment]
And this post about accessible data visualizations is great:  https://fossheim.io/writing/posts/accessible-dataviz-design/

**Comment by aleesteele** [comment]
Adding breadcrumbs to #2630
